src/js/index.js

```diff
@@ -0,0 +1,23 @@
+import '../scss/style.scss';
+import weatherApp from './weather';
+import * as view from './render';
+
+const weatherController = () => {
+  const searchParams = document.getElementById('search-bar');
+  const searchButton = document.querySelector('button');
+
+  view.renderPage();
+
+  const getLocation = () => {
+    const params = {
+      city: ((searchParams.value.match(/^(\w+ ?\w+[^,])/gi))[0]).toLowerCase(),
+      state: ((searchParams.value.match(/\w+[^, ]$/gi))[0]).toLowerCase(),
+    };
```

Comment:
Yeah, I just want to practice using regEx, they are really powerful. About the state, that was on my original request but then I noticed it wasn't necessary. I will change it for the country in case someone wants to look for a city with a repeated name.

Reply:
Ready! It can include the country